src/payment/strategies/paypal-commerce/paypal-commerce-funding-key-resolver.ts

```diff
@@ -18,7 +18,7 @@ export default class PaypalCommerceFundingKeyResolver {
                     return 'BANCONTACT';
                 case 'giropay':
                     return 'GIROPAY';
-                case 'przelewy24':
+                case 'p24':
```

Comment:
By description of this PR it should be a Venmo related change, what's the purpose of changing the above?

Reply:
I believe it is the fix for initialization of `p24` method (on the backend it has such code), but, I agree it makes sense to move such fixes into a separate PR

Reply:
It looks like github bug. there is no difference with master branch
https://github.com/bigcommerce/checkout-sdk-js/blob/master/src/payment/strategies/paypal-commerce/paypal-commerce-funding-key-resolver.ts#L21